Makefile

```diff
@@ -36,6 +36,14 @@ binary-arch-minimal: swagger
 binary-arch: swagger
 	env CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o bin/zot-$(ARCH) -tags extended,containers_image_openpgp -v -trimpath -ldflags "-X zotregistry.io/zot/pkg/api/config.Commit=${COMMIT} -X zotregistry.io/zot/pkg/api/config.BinaryType=extended -X zotregistry.io/zot/pkg/api/config.GoVersion=${GO_VERSION} -s -w" ./cmd/zot
 
+.PHONY: cli
+cli:
+	env CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o bin/zli -tags extended,containers_image_openpgp -v -trimpath -ldflags "-X zotregistry.io/zot/pkg/api/config.Commit=${COMMIT} -X zotregistry.io/zot/pkg/api/config.BinaryType=extended -X zotregistry.io/zot/pkg/api/config.GoVersion=${GO_VERSION} -s -w" ./cmd/zli
+
+.PHONY: cli-arch
+cli-arch: swagger
+	env CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) go build -o bin/zli-$(ARCH) -tags extended,containers_image_openpgp -v -trimpath -ldflags "-X zotregistry.io/zot/pkg/api/config.Commit=${COMMIT} -X zotregistry.io/zot/pkg/api/config.BinaryType=extended -X zotregistry.io/zot/pkg/api/config.GoVersion=${GO_VERSION} -s -w" ./cmd/zli
```

Comment:
Should we change GOARCH to arm64 instead of  default amd64?
What is strange to me is that we are using amd64 everywhere in this Makefile.
If I'm on Raspberry Pi  and run "make zot-arch" I expect the binary to work. The question is does it?

If the idea is that users will use it like "make ARCH=arm64 binary-arch-minimal && make ARCH=arm64 binary-arch" where in the documentation we described this?

Reply:
this is for ci/cd only, we are not doing any auto-detect of host platform